### PR TITLE
[ Relay ][ Frontend ][ Tensorflow ]add op add_n to relay/frontend/tensorflow.py

### PR DESCRIPTION
**Pull request by KimBioInfoStudio**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by soiferj** [comment]
LGTM

File: python/tvm/relay/frontend/tensorflow.py [hunk 1]
```diff
@@ -1323,6 +1335,7 @@ def _impl(inputs, attr, params):
 _convert_map = {
     'Abs'                               : AttrCvt('abs'),
     'Add'                               : _elemwise('add'),
+    'AddN'                              : _add_n(),
```
Comment (soiferj) [inline python/tvm/relay/frontend/tensorflow.py]: Please also add to docs [here](https://docs.tvm.ai/frontend/tensorflow.html).

File: tests/python/frontend/tensorflow/test_forward.py [hunk 2]
```diff
@@ -2357,6 +2357,28 @@ def test_forward_one_hot():
...
+# AddN
+# ----------------------
+def _test_forward_add_n(inputs):
+    tf.reset_default_graph()
+    with tf.Graph().as_default():
+        temp = []
+        for each in inputs:
+            temp.append(tf.placeholder(shape=each.shape, dtype=each.dtype))
+        output = tf.add_n(temp)
+        compare_tf_with_tvm([each for each in inputs],[each.name for each in inputs], output.name)
```
Comment (yongwww) [inline tests/python/frontend/tensorflow/test_forward.py]: CI failed due to numpy ndarray has no name attribute. I guess you meant `each.name for each in temp` here?

File: tests/python/frontend/tensorflow/test_forward.py [hunk 3]
```diff
@@ -2357,6 +2357,28 @@ def test_forward_one_hot():
...
+
+def test_forward_add_n():
+    x = np.random.randint(1, 100, size = (3,3,3), dtype = np.int32)
+    y = np.random.randint(1, 100, size = (3,3,3), dtype = np.int32)
```
Comment (yongwww) [inline tests/python/frontend/tensorflow/test_forward.py]: would be nice to add a test case to cover float32

File: tests/python/frontend/tensorflow/test_forward.py (hunk 2 above)
Comment (KimBioInfoStudio) [reply tests/python/frontend/tensorflow/test_forward.py]: yep, thx for ur suggestions

File: tests/python/frontend/tensorflow/test_forward.py (hunk 3 above)
Comment (KimBioInfoStudio) [reply tests/python/frontend/tensorflow/test_forward.py]: have add  to the test file

File: tests/python/frontend/tensorflow/test_forward.py [hunk 4]
```diff
@@ -2342,13 +2591,17 @@ def test_placeholder():
 #######################################################################
 # OneHot
 # ----------------------
+
```
Comment (soiferj) [inline tests/python/frontend/tensorflow/test_forward.py]: Why did all of these extra lines change?

File: tests/python/frontend/tensorflow/test_forward.py [hunk 5]
```diff
@@ -2357,6 +2610,40 @@ def test_forward_one_hot():
...
+    y = np.random.randint(1, 100, size=(3, 3, 3), dtype=np.int32)
+    z = np.random.randint(1, 100, size=(3, 3, 3), dtype=np.int32)
+    m, n, o = x.astype(np.float32), y.astype(np.float32), z.astype(np.float32)
+    in0 = x
+    in1 = [x, y]
+    in2 = (x, y, z)
+    in3 = m
+    in4 = [m, n]
+    in5 = (m, n, o)
+    _test_froward_add_n(in0)
```
Comment (yongwww) [inline tests/python/frontend/tensorflow/test_forward.py]:
```suggestion
    _test_forward_add_n(in0)
```

File: tests/python/frontend/tensorflow/test_forward.py (hunk 4 above)
Comment (KimBioInfoStudio) [reply tests/python/frontend/tensorflow/test_forward.py]: sorry for that problem will  fix it

File: python/tvm/relay/frontend/tensorflow.py (hunk 1 above)
Comment (soiferj) [reply python/tvm/relay/frontend/tensorflow.py]: @KimBioInfoStudio can you please add this to the docs and add a unit test?

**Comment by tqchen** [comment]
Thanks @soiferj @KimBioInfoStudio @yongwww !

**Pull request merged**